datasets/switchboard.py

```diff
@@ -90,7 +91,11 @@ def run(self):
                 ".",
             ]
         )
-        shutil.move("swb_ms98_transcriptions", self.out_trans_and_dict_dir)
+        shutil.move("swb_ms98_transcriptions", self.out_trans_dir.get_path())
+        shutil.copy(
```

Comment:
No need to copy the data, your second output path can also be within the swb_trans folder. E.g.:
```
self.out_raw_dict = self.output_path("swb_trans/sw-ms98-dict.text")
```

Reply:
but then what would be the default value for `self.out_raw_dict` in the constructor? `None` is ok?

Reply:
The path need to be defined beforehand right? so it should be some `self.output_path` already in the constructor.

Reply:
the suggestion is that after the move
```
shutil.move("swb_ms98_transcriptions", self.out_trans_dir.get_path())
```

there exists the file `swb_trans/sw-ms98-dict.text`
which you would copy to a new name. No need to do this if you define this name directly above because the file will then be used directly. (cf. code suggestion in other comment)

Reply:
ah i see